commands/certs/add.js

```diff
@@ -113,25 +113,69 @@ function * getChoices (certDomains, newDomains, existingDomains, context) {
   }
 }
 
+function * getDomains (context, heroku) {
+  function someNull (domains) {
+    return _.some(domains, (domain) => domain.kind === 'custom' && domain.cname == null)
```

Comment:
you could instead do

```js
domains.find(domain => domain.kind === 'custom' && !domain.cname)
```

I wouldn't ever write `== null`. It's exactly the same as `!` (I think), but more obvious you're referring to anything falsy.